modules/social_features/social_follow_taxonomy/modules/social_follow_landing_page/social_follow_landing_page.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Social Follow Landing Page'
+description: 'Provide follow functionality for landing pages.'
+type: module
+core: 8.x
+dependencies:
+  - social_follow_taxonomy:social_follow_taxonomy
+  - social_tagging:social_tagging
+  - social_landing_page:social_landing_page
+package: Social
```

Comment:
@denysyukm I think it's better to add it to Social (experimental) package

Reply:
DONE